### ISSUE TITLE
Add new bluetooth detectors

### ISSUE THREAD

**Issue by rickl4** [CityofToronto/bdit_data-sources#175]
I added the data to `rliu.new_detectors` which is in the exact same format as `bluetooth.detectors`.

For the excel spreadsheet, I'm still waiting on additional data from Quinn to fill in the missing fields.

**Comment by rickl4** [comment]
Segments are in, but not the detectors.

**Comment by radumas** [comment]
This has been superseded by #196
